_posts/2018/2018-02-05-enumerable-avoids-using-temporary-variables.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Enumerable avoids using temporary variables when looping'
+description: 'Idiomatic Ruby is in the looping'
+layout: article
+category: ruby
+image:
+  base: '2018/enumerable-avoids-using-temporary-variables'
+  alt: 'Fruit loops'
+  source: 'https://unsplash.com/photos/BumOnw4oEZo'
+  credit: 'David Streit'
+---
+
+Some of my favourite Ruby features are to be found in the Enumerable module. You can read more in the [Ruby documentation](http://ruby-doc.org/core-2.5.0/Enumerable.html).
+
+A class representing a collection, such as Array, Set or Hash, has the methods and features of Enumerable included. `Enumerable` contains methods to allow you to loop through the members of that group and take an action using each member of the collection as an input.
```

Comment:
Do you want to be doing `Enumerable` or just 'Enumerable'? You do both versions in this paragraph.

Reply:
And throughout the rest of the article. 

Is there a pattern to the differing usage? I can't spot one...

Reply:
There isn't. I might use italics?

Reply:
A class representing a collection, such as `Array`, `Set` or `Hash`, has the methods and features of `Enumerable` included. `Enumerable` contains methods to allow you to loop through the members of that group and take an action using each member of the collection as an input.

vs

A class representing a collection, such as _Array_, _Set_ or _Hash_, has the methods and features of _Enumerable_ included. _Enumerable_ contains methods to allow you to loop through the members of that group and take an action using each member of the collection as an input.

Reply:
What do you think @nodunayo ?

Reply:
I think back-ticks, rather than italics.